regression/cbmc-java/inherited_static_field10/Test.java

```diff
@@ -0,0 +1,10 @@
+public class Test extends Parent {
+
+  public static void main(int nondet) {
```

Comment:
I did not know it is valid Java to have a static main method that doesn't take `String[]` - awesome.

Reply:
It isn't in Java, but it is in jbmc :) The JVM would say "that's not a proper main()", but jbmc only cares about the name, not the arg types.

Reply:
Not blocking - but I'd suggest this isn't a great thing to therefore rely on in tests...

Reply:
With #1768, you'll have to provide a --function to make this test run.